superset/app.py

```diff
@@ -17,8 +17,7 @@
 
 import logging
 import os
-from typing import cast, Iterable, Optional
-from wsgiref.types import StartResponse, WSGIApplication, WSGIEnvironment
```

Comment:
wsgiref.types does not exist on Python 3.10

Reply:
fixing from https://github.com/apache/superset/commit/09b92e7d08bc96b6006938c00d9f8ce7a108102b